tests/test_es_doc_publisher.py

```diff
@@ -244,7 +244,7 @@ class Test_Unit_Tests:
             ("-test_name", "test_name"),
             ("test/name", "testname"),
             ("test? name", "test_name"),
-            ("Test_Name", "test_name"),
```

Comment:
The `Test_Name` snapshot was colliding on my filesystem with the `test_name` snapshot. I renamed this to avoid case-insensitive file system issues.

Reply:
Thank you!